app/controllers/tasks_controller.rb

```diff
@@ -10,14 +10,26 @@ def create
     redirect_to list_path(@list)
   end
 
+  def show
+    @task = Task.find(params[:id])
```

Comment:
What do you think about breaking `Task.find(params[:id])` into a [`filter`](http://guides.rubyonrails.org/action_controller_overview.html#filters) or private method to help reduce the repetition?